language/move-lang/tests/move_check/translated_ir_tests/generics/instantiation_loops/mutually_recursive_three_args_type_con_shifting.exp

```diff
@@ -0,0 +1,38 @@
+error: 
+
+    ┌── tests/move_check/translated_ir_tests/generics/instantiation_loops/mutually_recursive_three_args_type_con_shifting.move:9:9 ───
+    │
+  9 │         h<T1, T2, S<T3>>()
+    │         ^^^^^^^^^^^^^^^^^^ Invalid call to '0x8675309::M::h'
+    ·
+  9 │         h<T1, T2, S<T3>>()
+    │                   ----- The type parameter 'f::T3' was instantiated with the type '0x8675309::M::S<T3>', which contains the type parameter 'h::T3'. A cycle of recursive calls causes that instantiation to recurse infinitely
```

Comment:
The wording "A cycle of recursive calls causes that instantiation" seems a bit awkward. What do   you think of changing the "that" to either "this" or "the"?